narwhals/functions.py

```diff
@@ -329,91 +328,56 @@ def from_dict(
         |     1  2  4      |
         └──────────────────┘
     """
-    return _from_dict_impl(data, schema, backend=backend)
+    return _from_dict_impl(data, schema, backend=backend, version=Version.V1)
```

Comment:
Should be main namespace here?

```suggestion
    return _from_dict_impl(data, schema, backend=backend, version=Version.MAIN)
```

Reply:
Oooh well spotted!